corehq/apps/reports/static/reports/v2/js/datagrid.js

```diff
@@ -127,6 +127,20 @@ hqDefine('reports/v2/js/datagrid', [
             self.data.refresh();
         };
 
+        self.unsupportedTaskName = ko.observable();
```

Comment:
I think you could kill this code and still have a perfectly reasonable user experience.

Reply:
On the other hand, it would be worth differentiating between the two actions if we're going to collect analytics on how often this gets clicked - could/should we do that?

Reply:
We are going to collect analytics on who clicked what. just haven't set it up yet